framework/src/main/java/org/apache/drill/test/framework/TestDriver.java

```diff
@@ -215,6 +212,14 @@ public void generateReports(List<DrillTest> tests, int iteration) {
       }
       bufferedWriter.flush();
       bufferedWriter.close();
+
+      // Upload report to DFS if the drillReportsDFSDir variable is set
+      if (!drillReportsDFSDir.isEmpty()){
```

Comment:
I had the same question before. If drillReportsDFSDir is defined, we will have two copies of the report files. My question is is it possible to have only one copy? Imagine I only want to getup my test environment to keep one copy of the report on DFS. Is it possible?